src/main/java/org/sagebionetworks/web/client/EntityTypeUtils.java

```diff
@@ -123,6 +130,9 @@ public static IconType getIconTypeForEntityClassName(String className) {
 		} else if (EntityView.class.getName().equals(className)) {
 			// FileView
 			icon = IconType.TH_LIST;
+		} else if (MaterializedView.class.getName().equals(className)) {
+			// FileView
+			icon = IconType.TH_LIST;
```

Comment:
Is this dead code?

Reply:
Looks like it's still used by the Provenance graph but that's it